### PR TITLE
Fix crash with 3DLUT feature

### PR DESCRIPTION
**Pull request by shun-iwasawa**
#1862 

Since the global shared OpenGL context is disabled, I made 3DLUT-related resources (`LutCalibrator`) to be held by each GL widget individually.
The LUT array data itself is loaded once and held in `LutManager` in order to share it between widgets.

### CODE REVIEW

**Comment by shun-iwasawa** [comment]
Jenkins

**Comment by Orphanlast** [comment]
Can I get this in layman's terms in some way so I can report this story in an Opentoonz News video?

**Comment by masafumi-inoue** [comment]
LGTM

**Comment by Orphanlast** [comment]
Okay, I'm removing this story from the news. This sounds interesting but if I have no way of reporting the news on it. I'll ignore the story. Furthermore, once I delete the link from my records, I'll probably not find this page again. So, if someone decides to explain this as being news worthy, don't bother, I probably won't read it. It's just the way things go I guess.

**Comment by shun-iwasawa** [comment]
@Orphanlast
I'm just wondering if you have read [my previous comment](https://github.com/opentoonz/opentoonz/issues/1862#issuecomment-372580446) on #1862 .

**Comment by jpturcotte** [comment]
@Orphanlast Please pay attention to the language you use so as to not come off as rude. Remember that the maintainers do not owe us anything.
